src/azure-cli/azure/cli/command_modules/network/tests/latest/recordings/test_private_link_resource_acfr.yaml

```diff
@@ -0,0 +1,153 @@
+interactions:
+- request:
+    body: '{"location": "eastus", "properties": {"tenantSettings": {}, "sku": {"name":
+      "Basic", "family": "c", "capacity": 0}}}'
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      CommandName:
+      - redis create
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '116'
+      Content-Type:
+      - application/json
+      ParameterSetName:
+      - --location --name --resource-group --sku --vm-size
+      User-Agent:
+      - AZURECLI/2.29.1 azsdk-python-mgmt-redis/13.0.0 Python/3.9.7 (Windows-10-10.0.19043-SP0)
+    method: PUT
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/cli_test_acfr_plr000001/providers/Microsoft.Cache/redis/cli-test-acfr-plr000002?api-version=2020-12-01
+  response:
+    body:
+      string: '{"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/cli_test_acfr_plr000001/providers/Microsoft.Cache/Redis/cli-test-acfr-plr000002","location":"East
+        US","name":"cli-test-acfr-plr000002","type":"Microsoft.Cache/Redis","tags":{},"properties":{"provisioningState":"Creating","redisVersion":"4.0.14","sku":{"name":"Basic","family":"C","capacity":0},"enableNonSslPort":false,"instances":[{"sslPort":15000,"isMaster":false,"isPrimary":false}],"publicNetworkAccess":"Enabled","tenantSettings":{},"redisConfiguration":{"maxclients":"256","maxmemory-reserved":"2","maxfragmentationmemory-reserved":"12","maxmemory-delta":"2"},"accessKeys":{"primaryKey":"<primaryKey>","secondaryKey":"<secondaryKey>"},"hostName":"cli-test-acfr-plr000002.redis.cache.windows.net","port":6379,"sslPort":6380,"linkedServers":[]}}'
```

Comment:
Manually editing YAML to replace credential is not a good practice. Credential Scan will fail again once regenerated.